paimon-core/src/main/java/org/apache/paimon/catalog/Catalog.java

```diff
@@ -258,6 +258,18 @@ default boolean caseSensitive() {
         return true;
     }
 
+    default void repairCatalog() throws DatabaseNotExistException {
+        throw new UnsupportedOperationException();
+    }
+
+    default void repairDatabase(String databaseName) throws DatabaseNotExistException {
+        throw new UnsupportedOperationException();
+    }
+
+    default void repairTable(Identifier identifier) throws TableNotExistException {
```

Comment:
Add some comments.

Reply:
Querying the paimon table by identifier in the file system may not exist. `TableNotExistException` is the exception thrown on the getDataTable method.